### PR TITLE
Add support info to self hosted page

### PR DESCRIPTION
**Pull request by kathweinschenkprophecy**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208947658816309

### CODE REVIEW

File: docs/architecture/self-hosted/self-hosted.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ There are a couple of ways to install the self-managed version of Prophecy:
 
 Once Prophecy is installed, you'll have to manually perform upgrades, backups, restores, etc.
 
+:::info
+For personalized help, contact [support](https://prophecy.zendesk.com/). If applicable, send along your [logs](download-logs.md) to help us troubleshoot.
+:::
+
```
Comment (atbida) [inline docs/architecture/self-hosted/self-hosted.md:27]: Please add in this note that the "Download Logs" feature is enabled by default on Prophecy v3.4.1.0+

File: docs/architecture/self-hosted/download-logs.md [hunk 2]
```diff
@@ -8,12 +8,9 @@ tags:
   - support logs
 ---
 
-As an admin user, you can download your environment logs from Prophecy without needing access to your Prophecy cluster or the assistance of Prophecy Support. This reduces the delay in debugging any issues with your Prophecy services.
+Prophecy lets you download logs to help debug issues with your Prophecy services. As an admin, you can download environment logs directly from Prophecy without needing access to your Prophecy cluster or assistance from Prophecy Support.
```
Comment (atbida) [inline docs/architecture/self-hosted/download-logs.md:12]: @kathweinschenkprophecy would you mind copying this image to use here as well? https://docs.prophecy.io/release_notes/2024/November_2024/#download-logs

I know we don't like to duplicate content, but a picture really is needed here.

Comment (kathweinschenkprophecy) [reply docs/architecture/self-hosted/download-logs.md:12]: yes I'll just direct it to that same image!

File: docs/architecture/self-hosted/self-hosted.md (hunk 1 above)
Comment (kathweinschenkprophecy) [reply docs/architecture/self-hosted/self-hosted.md:27]: will do

Comment (kathweinschenkprophecy) [reply docs/architecture/self-hosted/self-hosted.md:27]: I'll add it to the download logs page, assuming they have to go there anyway to learn about how to use it

Comment (atbida) [reply docs/architecture/self-hosted/self-hosted.md:27]: @kathweinschenkprophecy I like the mention of 3.4.1.0+, however I'm not seeing **by default**

Maybe we should remove the steps to enable the feature. I'll ping in #customer-success channel to ask on this.

**Review by atbida — APPROVED** [review]
lgtm, we can decide if we need further updates by the response to this question: 
https://simpledatalabs.slack.com/archives/C02QPQS3AQ6/p1734105102941479

**Pull request merged**